example/imagery/plugin.js

```diff
@@ -168,7 +168,7 @@ function getImageUrlListFromConfig(configuration) {
 }
 
 function getImageLoadDelay(domainObject) {
-    const imageLoadDelay = domainObject.configuration.imageLoadDelayInMilliSeconds;
+    const imageLoadDelay = parseInt(domainObject.configuration.imageLoadDelayInMilliSeconds, 10);
```

Comment:
```suggestion
    const imageLoadDelay = parseInt(domainObject.configuration.imageLoadDelayInMilliSeconds);
```

Nice catch! We can omit the radix from the `parseInt()` call as it assumes decimal by default. This is nice because it forces the `imageLoadDelay` to be an integer value. The only (negligible?) issue I can see here is that E-notation values such as `2e3` won't parse correctly.